README.md

```diff
@@ -120,3 +130,32 @@ Be careful with this method of deployment. `bootRun` is not designed to work wit
 ## External
 
 Deploy resultant `cas/build/libs/cas.war` to a servlet container of choice.
+
+# DEBUG
+
+## Build
+
+```bash
+# shortcut for clean, build and explodeWar (mainly for debug purposes)
+./gradlew[.bat] buildit
```

Comment:
The bash script acts as a bridge to simplify the gradle commands; it is not a replacement. It is so that you can run "build.sh debug". It is the shortcut, and is to keep things possibly more consistent between the projects. Bash arguably is easier for folks to deal with, as "everyone knows how to run bash script" and can decipher it, rather than running "gradlew builtit" or anything similar.